Project-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,12 @@
 import React from 'react';
 import './Card.css';
+
+const CardBanner = props => {
+	return (
+		<div className="cardBanner">
+			<img src={require("../images/calculator-component.png")} id="mid-img"/>
```

Comment:
require is old React syntax...I would use import and import the image under your `import './Card.css'` line. 

`import Logo from '../images/nameoffile.png';`